projects/proxy.yaml

```diff
@@ -0,0 +1,9 @@
+paths:
```

Comment:
Why is this file in the `projects` dir? It's not really a project...

Reply:
it's kinda a collection of modules which all the projects are..